src/main/resources/assets/nei/csv/handlers.csv

```diff
@@ -451,3 +451,18 @@ neresources.nei.NEIMobHandler,Not Enough Resources,minecraft:skull:4,,neresource
 neresources.nei.NEIOreHandler,Not Enough Resources,minecraft:diamond_ore,,neresources,TRUE,,,65,166,5,,,,,,
 neresources.nei.NEIPlantHandler,Not Enough Resources,minecraft:wheat,,neresources,TRUE,,,130,166,2,,,,,,
 wanion.biggercraftingtables.nei.BigShapedRecipeHandler,Bigger Crafting Tables,minecraft:crafting_table,,biggercraftingtables,TRUE,,,130,,2,,,,,,
+net.machinemuse.numina.nei.JSONRecipeHandler,Modular Powersuits,powersuits:tile.tinkerTable,,powersuits,TRUE,,,65,166,5,,,,,,
```

Comment:
I think the normal crafting table should be used here because the recipes displayed here must be crafted in one.

Reply:
I sort of disagree. While I understand they're just another crafting table recipes, it is good to recognize and use to find parts related to MPS you wouldn't find it normally.

Same reason as IC2 recipes have a different icon.